resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/features/BasicDriverFeatureStepSuite.scala

```diff
@@ -35,7 +35,9 @@ import org.apache.spark.util.Utils
 
 class BasicDriverFeatureStepSuite extends SparkFunSuite {
 
-  private val CUSTOM_DRIVER_LABELS = Map("labelkey" -> "labelvalue")
+  private val CUSTOM_DRIVER_LABELS = Map(
+    "labelkey" -> "labelvalue",
+    "yunikorn.apache.org/app-id" -> "{{APPID}}")
```

Comment:
Although I understand the intention, let's isolate `yunikorn` related feature inside `YuniKornSuite.scala`.

Reply:
Previously, we mixed that and it turns that it was not good in a long term perspective.